modules/openid_auth/config/routes.rb

```diff
@@ -6,6 +6,7 @@
   namespace :v0, defaults: { format: 'json' } do
     get 'validation', to: 'validation#index'
     get 'mvi-user', to: 'mvi_users#show'
+    post 'mvi-user', to: 'mvi_users#create'
```

Comment:
I would use a custom action name in order to avoid the inferrence that this is actually creating a new resource. To me we're relying on the POST method for this purpose in [the HTTP spec](https://tools.ietf.org/html/rfc7231#section-4.3.3):

> Providing a block of data, such as the fields entered into an HTML form, to a data-handling process;

Whereas the rails REST routing convention is aimed primarily at this purpose:

> Creating a new resource that has yet to be identified by the origin server

It's probably worth changing the path for the POST request to identify the _"data handling process"_. Maybe `mvi-user-lookup`?